### PR TITLE
fix(repo): babel + miscs

### PR DESCRIPTION
**Pull request by kevinadda**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -35,6 +35,10 @@ module.exports = {
               'transform-react-jsx',
               'react-hot-loader/babel',
               'transform-runtime',
+              'transform-class-properties',
+              'babel-root-import',
+              'transform-async-to-generator',
+              'syntax-async-functions',
```
Comment (windkomo) [inline webpack.config.js]: Please copy those changes to webpack.prod.config.js too

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (windkomo) [inline .babelrc]: You can remove this file because this is configured in webpack

File: package.json [hunk 3]
```diff
@@ -13,8 +13,10 @@
   "devDependencies": {
     "babel-cli": "^6.24.1",
     "babel-eslint": "^7.2.3",
+    "babel-plugin-module-resolver": "^2.7.1",
```
Comment (windkomo) [inline package.json]: This should be removed (unused in webpack.config)

File: webpack.config.js (hunk 1 above)
Comment (kevinadda) [reply webpack.config.js]: ack

File: .babelrc (hunk 2 above)
Comment (kevinadda) [reply .babelrc]: We did not

File: package.json (hunk 3 above)
Comment (kevinadda) [reply package.json]: it is for root path resolving

**Pull request merged**